Sources/StreamChatUI/Utils/UIImage+Extensions.swift

```diff
@@ -10,6 +10,21 @@ extension UIImage {
     }
 }
 
+extension UIImage {
+    func circleImage() -> UIImage {
+        let size: CGSize = CGSize(width: 24, height: 24)
+        let renderer = UIGraphicsImageRenderer(size: size)
+        let circleImage = renderer.image { ctx in
+            ctx.cgContext.setFillColor(UIColor.red.cgColor)
+
+            let rectangle = CGRect(x: 0, y: 0, width: size.width, height: size.height)
+            ctx.cgContext.addEllipse(in: rectangle)
+            ctx.cgContext.drawPath(using: .fillStroke)
+        }
+        return circleImage
+    }
```

Comment:
Do you think we can go with `static let circle: UIImage = {}()` so the image is rendered just once?

Reply:
Great idea @evsaev made that change now 👍